context/app/static/js/components/genes/Summary/KnownReferences.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
+import hasKey from 'js/helpers/hasKey';
+import { capitalizeString } from 'js/helpers/functions';
+
+import { Skeleton } from '@mui/material';
+import { useGeneDetails } from '../hooks';
+import { ReferenceList } from './styles';
+
+// Map reference sources to display names
+// default to capitalizing the original source name if not found
+const referenceMap = new Proxy<Record<string, string>>(
+  {
+    entrez: 'Entrez',
+    hugo: 'HUGO HGNC',
+    omim: 'OMIM',
+    uniprotkb: 'UniProt',
+    ensembl: 'Ensembl',
+  },
+  {
+    get: (target: object, prop: string) => (hasKey(target, prop) ? target[prop] : capitalizeString(prop)),
+  },
+);
+
+function ReferenceListContents() {
+  const { data } = useGeneDetails();
+  if (!data) {
+    return (
+      <>
+        <Skeleton width={100} />
+        <Skeleton width={120} />
+        <Skeleton width={80} />
+      </>
+    );
+  }
+  return (
+    <>
+      {data.references.map(({ url, source }) => (
+        <dd key={source}>
+          <OutboundIconLink href={url}>{referenceMap[source]}</OutboundIconLink>
+        </dd>
```

Comment:
Apologies if I'm missing something, but `dl` requires both a `dt` and `dd` for each item? https://www.w3.org/TR/html401/struct/lists.html#h-10.3

Reply:
You're completely right; I've previously used dt/dd pairs to display metadata key/value pairs, but they're not suited for this references list since the source and link are presented as one item. A `ul` will be the better option here.